webodf/lib/gui/SessionController.js

```diff
@@ -116,6 +116,13 @@ gui.SessionControllerOptions = function () {
             "Expected to be run in an environment which has a global window, like a browser.");
 
         /**
+         * @return {!gui.SessionControllerOptions}
+         */
+        this.getConfiguration = function() {
```

Comment:
Agree with removing. I believe I was using it at one stage to avoid disabled controls... not necessary at the moment.